os/drivers/watchdog.c

```diff
@@ -274,6 +274,9 @@ static int wdog_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 	DEBUGASSERT(upper && lower);
 
 	/* Get exclusive access to the device structures */
+	if (cmd < 0) {
```

Comment:
This is not matched with comment. Please move the comment.

Reply:
yes

Reply:
i did it